### PR TITLE
feat: deployment script using starkli

### PR DESCRIPTION
**Pull request by sparqet**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Feature

## What is the current behavior?

Resolves: #NA

## What is the new behavior?

- deployment script for role_store
-
-

## Does this introduce a breaking change?

 No

### CODE REVIEW

File: scripts/deploy_role_store_contract.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Deployment script for role_store.cairo
+
+command_output=$(starkli declare ../target/dev/satoru_RoleStore.sierra.json)
```
Comment (Sk8erboi84) [inline scripts/deploy_role_store_contract.sh:5]: Explain each line

File: scripts/deploy_role_store_contract.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+from_char=":"
```
Comment (Sk8erboi84) [inline scripts/deploy_role_store_contract.sh:8]: Explain each line

File: scripts/deploy_role_store_contract.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+class_hash=$(echo "$command_output" | sed 's/.*'$from_char'//')
```
Comment (Sk8erboi84) [inline scripts/deploy_role_store_contract.sh:10]: Explain each line

File: scripts/deploy_role_store_contract.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+
+starkli deploy $class_hash
```
Comment (Sk8erboi84) [inline scripts/deploy_role_store_contract.sh:13]: Explain each line

**Review by Sk8erboi84 — CHANGES_REQUESTED** [review]

**Review by zarboq — APPROVED** [review]

**Comment by Sk8erboi84** [comment]
Please change the issue and remove sncast and add starkli

**Review by Sk8erboi84 — APPROVED** [review]

**Pull request merged**